src/openrct2/world/Sprite.h

```diff
@@ -13,97 +13,17 @@
 #include "../common.h"
 #include "../peep/Peep.h"
 #include "../ride/Vehicle.h"
+#include "Balloon.h"
+#include "Duck.h"
 #include "Entity.h"
 #include "Fountain.h"
+#include "Litter.h"
+#include "MoneyEffect.h"
 #include "SpriteBase.h"
+#include "Particle.h"
```

Comment:
```suggestion
#include "Particle.h"
#include "SpriteBase.h"
```
Just to fix formatting and get the CI going